2018/readme.md

```diff
@@ -0,0 +1,128 @@
+
+## Configurations 
+The following yaml configuration files and values can be changed to alter the output of HMDA submission test files. 
+
+### [Test Files Configuration](https://github.com/cfpb/hmda-test-files/blob/master/2018/python/configurations/test_filepaths.yaml)
+
+| Key| Default Value | Expanation  |
+|:-------------|:-------------|:--------|
+|clean_filepath | "../edits_files/clean_files/{bank_name}/" | Filepath for generated clean files. Note: the filepath needs to contain `{bank_name}` for formatting the institution name to the path. |
+| clean_filename|"clean_file_{n}_rows_{bank_name}.txt"| Filename for generated clean files. Note: the filepath needs to contain `{bank_name}` and `{n}` for formatting the institution name and number of rows to the filename. |
+| validity_filepath | "../edits_files/test_files/{bank_name}/validity/"| Filepath for generated validity edit test files. Note: the filepath needs to contain `{bank_name}` for formatting the institution name to the path.|
+|syntax_filepath |"../edits_files/test_files/{bank_name}/syntax/"|Filepath for generated syntax edit test files. Note: the filepath needs to contain `{bank_name}` for formatting the institution name to the path. 
+|quality_filepath | "../edits_files/test_files/{bank_name}/quality/"|Filepath for generated quality edit test files. Note: the filepath needs to contain `{bank_name}` for formatting the institution name to the path. 
+| quality_pass_s_v_filepath|"../edits_files/test_files/{bank_name}/quality_pass_s_v/"| Filepath for generated quality edit test files that also pass syntax and validity edits. Note: the filepath needs to contain `{bank_name}` for formatting the institution name to the path |
+ts_schema_json | "../schemas/ts_schema.json"| Filename for the Transmital Sheet schema. 
+lar_schema_json| "../schemas/lar_schema.json"| Filename for the Loan Application Register schema. |
+edit_report_filepath| "../edits_files/"|Filepath for storing generated edit reports.|
+edit_report_filename| "edit_report.csv"|Filename for generated edit reports.|
+log_filename | "clean_files_log.txt" | Filename for the log file detailing iterations in the file creation process.|
+log_mode| 'a'| The mode for logging files. Currently, the log mode is set to append log entries ('a'). This configuration can be changed to write over the previous file ('w').
+
+
+### [Geographic Data Configuration](https://github.com/cfpb/hmda-test-files/blob/master/2018/python/configurations/geographic_data.yaml)
+
+| Key| Default Value | Expanation  |
+|:-------------|:-------------|:--------|
+|crosswalk_data_file | "../dependencies/census_2018_MSAMD_name.txt" | Filepath and name for geographic cross walk data located in the [dependencies folder](https://github.com/cfpb/hmda-test-files/blob/master/2018/dependencies/). The crosswalk data file contains relationships between variables such as state, county, census tract, MSA, and population that are used to generate clean files and edit files. |
```

Comment:
Please change the quote characters to ` to render the default value as code pieces. 

If you think a data type indication is needed, please add a data type column and list the type there.

Reply:
Sure. I can do this.